### PR TITLE
ENCD-6091-add-Hongkai-Ji-submission-properties

### PR DESCRIPTION
**Pull request by iyoungworth**
corrected-step-label

fixture-correction

another-fixture-adjustment

### CODE REVIEW

File: src/encoded/schemas/file.json [hunk 1]
```diff
@@ -1618,6 +1618,7 @@
                         "fine-mapped variants",
                         "fold change over control",
                         "footprints",
+                        "functional conservation scores",
```
Comment (gabdank) [inline src/encoded/schemas/file.json]: can we change it to "functional conservation quantifications"?

File: src/encoded/schemas/analysis_step.json [hunk 2]
```diff
@@ -342,6 +343,7 @@
                     "fine-mapped variants",
                     "fold change over control",
                     "footprints",
+                    "functional conservation scores",
```
Comment (gabdank) [inline src/encoded/schemas/analysis_step.json]: can we change it to "functional conservation quantifications"?

File: src/encoded/schemas/analysis_step.json [hunk 3]
```diff
@@ -154,6 +154,7 @@
...
```
Comment (gabdank) [inline src/encoded/schemas/analysis_step.json]: can we change it to "functional conservation quantifications"?

File: src/encoded/tests/data/inserts/file.json [hunk 4]
```diff
@@ -4709,5 +4709,24 @@
...
+        "md5sum": "3904fa99de87d37fc36bfa282c349cdb",
+        "accession": "ENCFF000CFC",
+        "file_format": "bed",
+        "file_format_type": "bed9",
+        "award": "/awards/U54HG006998/",
+        "assembly": "GRCh38",
+        "genome_annotation": "V19",
+        "dataset": "/experiments/ENCSR000CFC/",
+        "file_size": 370148294,
+        "output_type": "functional conservation scores",
```
Comment (gabdank) [inline src/encoded/tests/data/inserts/file.json]: can we change it to "functional conservation quantifications"?

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/encoded/schemas/file.json (hunk 1 above)
Comment (iyoungworth) [reply src/encoded/schemas/file.json]: Changed

File: src/encoded/schemas/analysis_step.json (hunk 2 above)
Comment (iyoungworth) [reply src/encoded/schemas/analysis_step.json]: Changed

File: src/encoded/schemas/analysis_step.json (hunk 3 above)
Comment (iyoungworth) [reply src/encoded/schemas/analysis_step.json]: Changed

File: src/encoded/tests/data/inserts/file.json (hunk 4 above)
Comment (iyoungworth) [reply src/encoded/tests/data/inserts/file.json]: Changed

**Review by gabdank — APPROVED** [review]

**Pull request merged**